db/schema.rb

```diff
@@ -222,9 +222,9 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "payment_type", null: false
-    t.integer "amount", null: false
-    t.string "frequency", null: false
-    t.jsonb "metadata", default: {}, null: false
+    t.integer "amount"
+    t.string "frequency"
+    t.jsonb "metadata", default: {}
```

Comment:
This schema change to remove the `not null` property will need a dedicated migration because the existing staging/production database already enforces null: false. Otherwise your user journey will fail on staging/production